### DISCUSSION TITLE
DLL“onnxruntime “   OrtSessionOptionsAppendExecutionProvider_DML Unable to find entry point

### DISCUSSION THREAD

**Discussion by Ann-Jack** [microsoft/onnxruntime#14491] [Other Q&A]
Excuse me?

1、nuget OnnxRuntime 1.13.1
2、Net framework 4.7.2 
3、No problem with use cup inference。 
 
When I use DML acceleration, I can't find the entry point（ options.AppendExecutionProvider_DML(1);） `
`

**Comment by RyanUnderhill** [comment]
Be sure to get the DirectML specific package, the CPU only package will not have it:

https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.DirectML
